.flake8

```diff
@@ -13,7 +13,8 @@ per-file-ignores =
   # Y023: Alias typing as typing_extensions
   # Y026: Have implicit type aliases
   # Y053: have literals >50 characters long
-  *_pb2.pyi: Y021, Y023, Y026, Y053
+  # Y065: TODO: Remove bare Incompletes
+  *_pb2.pyi: Y021, Y023, Y026, Y053, Y065
```

Comment:
This only ignores Y065 for files ending in `_pb2.pyi`

Reply:
That happens when editing too many branches at the same time ...